modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -240,3 +240,79 @@ If Kuryr uses `ovn` instead of Amphora, it offers the following benefits:
 * Reduced network latency.
 * Increased service creation speed by using OpenFlow rules instead of a VM for each Service.
 * Distributed load balancing actions across all nodes instead of centralized on Amphora VMs.
+
+
+[id="installation-osp-kuryr-octavia-driver_upgrade_{context}"]
+== Upgrading to Octavia OVN Driver on existing deployments
+
+It is possible to upgrade {product-title} that has been deployed before OVN Octavia driver was made available on the {rh-openstack} side.
+The steps to perform the upgrade from amphora to ovn-octavia are the next:
+
+[NOTE]
+====
+Upgrading from amphora to ovn-octavia loadbalancer requires recreation of the
+loadbalancers, which means there will be some downtime when accessing the pods
+through their associated services -- time needed to delete the amphora load
+balancer plus the time needed to create the ovn load balancer.
+====
+
+.Procedure
+
+. Check ovn-octavia driver is available
++
+----
+$ openstack loadbalancer provider list
++---------+-------------------------------------------------+
+| name    | description                                     |
++---------+-------------------------------------------------+
+| amphora | The Octavia Amphora driver.                     |
+| octavia | Deprecated alias of the Octavia Amphora driver. |
+| ovn     | Octavia OVN driver.                             |
++---------+-------------------------------------------------+
+----
+
+. Edit the `kuryr-config` ConfigMap to trigger the recreation. You simply need
+to delete the annotation that states the `default` kuryr-octavia-provider is
+being used:
++
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+...
+kind: ConfigMap
+metadata:
+  annotations:
+    networkoperator.openshift.io/kuryr-octavia-provider: default <-- Remove this line
+...
+----
+
+. Wait until Cluster Network Operator reconcile loop is executed and detects the
```

Comment:
What is a specific thing that happens to indicate that these things have occurred?

Reply:
you can watch the cluster network operator pod logs to see when that happens. Otherwise you can simply check that the pods on the openshift-kuryr namespace will be recreated